packages/shared/trade/src/maturity-select/components/maturities.tsx

```diff
@@ -18,6 +18,11 @@ export function Maturities({
   inputLabel,
 }: MaturitiesProps) {
   const theme = useTheme();
+
+  if (!selectedfCashId) {
```

Comment:
I guess we need to discuss the logic around this, not sure if we should select the longest dated maturity by default